hash_writer_test.go

```diff
@@ -69,15 +164,14 @@ func TestWriteNull(t *testing.T) {
 	assert.NoError(t, ionHashWriter.Finish(), "Something went wrong executing ionHashWriter.Finish()")
 
 	// We're comparing splits because str.String() uses a cumbersome '\n' separator
-	expected := strings.Split("null null.float null.blob null.struct ", " ")
+	expected := strings.Split("null null.null null.bool null.int null.float null.decimal null.timestamp "+
+		"null.symbol null.string null.clob null.blob null.list null.sexp null.struct ", " ")
```

Comment:
Could we look into making this a parameterized test (this looks simple enough as an approach and takes no dependencies https://gist.github.com/zaky/8304b5abbdc03600c29e) each iteration will create a new writer and tackle 1 use case (1 input, 1 output, and hopefully no need for string splitting). And we can use this parameterized pattern for the other Ion types and their cases.

Reply:
Done.

Reply:
Fantastic. Thank you!